index.js

```diff
@@ -13,6 +14,9 @@ module.exports = function (browserify, options) {
     throw new Error('css-modulesify needs the --output / -o option (path to output css file)');
   }
 
+  // PostCSS plugins passed to FileSystemLoader
+  var plugins = options.plugins || options.p || Core.defaultPlugins;
```

Comment:
@joshgillies do you know if `plugins` works as strings or does `css-modules-loader-core` expect an array of objects?